Sources/SwiftToolchain/Toolchain.swift

```diff
@@ -283,21 +283,21 @@ public final class Toolchain {
     return testBundlePath
   }
 
-  public func packageInit(name: String, type: PackageType, inPlace: Bool) throws {
-    var initArgs = [
+  public func runPackageInit(name: String, type: PackageType, initialiseInSubdirectory: Bool) throws {
+    var arguments = [
       swiftPath.pathString, "package", "init",
       "--type", type.rawValue,
     ]
-    if !inPlace {
-      initArgs.append(contentsOf: ["--name", name])
-    }
-    try ProcessRunner(initArgs, terminal)
-      .waitUntilFinished()
+	
+	if initialiseInSubdirectory {
+		arguments += ["--name", name]
+	}
```

Comment:
Formatting here looks weird. Could you run `swiftformat` on the whole project please? If you run `pre-commit install` [as described in `README.md`](https://github.com/swiftwasm/carton#coding-style) this will install a pre-commit hook that will run SwiftLint and SwiftFormat before you commit.

Reply:
I'll run `swiftformat` and `pre-commit install`. I didn't read that part of `README` previously.

Reply:
Thanks, and no problem at all 🙂